README.md

```diff
@@ -23,6 +23,30 @@ computers.
   </a>
 </div>
 
+## Installation
+
+See [CLI](./cli/README.md).
+
+## Basic use
+
+```js
+while (true) {
+	const start = new Date();
+	const res = await fetch("https://ipfs.io/ipfs/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi/");
+	console.log(`TTFB: ${new Date() - start}ms`);
+}
```

Comment:
Please add a bit of delay so to avoid excessive CPU & network usage.

```suggestion
  await new Promise(resolve => setTimeout(resolve, 1000));
}
```

(I tested your code snippet with my change to verify that we can use top-level await.)